shared/chat/conversation/info-panel/container.tsx

```diff
@@ -15,6 +16,17 @@ type OwnProps = {
   conversationIDKey: Types.ConversationIDKey
   onBack?: () => void
   onCancel?: () => void
+  onSelectTab: (string) => void
+  selectedTab: string | null
+  onSelectAttachmentView: (typ: RPCChatTypes.GalleryItemTyp) => void
+  selectedAttachmentView: RPCChatTypes.GalleryItemTyp
+}
+
+const getFromMsgID = info => {
```

Comment:
Consider typing this argument \o/

Reply:
get what? consider also adding a return type!

Reply:
"FromMsgID" is the thing being got. It is the parameter to the RPC, meaning "load all things from this message ID into the past"